src/language-html/parser-html.js

```diff
@@ -288,6 +339,11 @@ module.exports = {
     vue: createParser({
       recognizeSelfClosing: true,
       isTagNameCaseSensitive: true,
+      getTagContentType: (tagname) => {
+        if (tagname !== "template") {
```

Comment:
```suggestion
        if (tagname.toLowerCase() !== "template") {
```

Reply:
I'm not sure if this is necessary. I haven't seen template tag written with upper case. I also can't find any such mention in [the specification](https://vue-loader.vuejs.org/spec.html) or in the source code of the [sfc-compiler](https://github.com/vuejs/vue-next/tree/master/packages/compiler-sfc).